postHandler.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
+const querystring = require('querystring');
+const makeTemplate = require('./elementTemplate');
+const updateIndex = require('./indexTemplate');
+
+const postHandler = (req, res) => {
+  console.log(`req.url is: ${req.url}`);
+
+  let body = {};
+
+  req.on('data', (chunk) => {
+    const queriedChunk = querystring.parse(chunk.toString());
+
+    body = queriedChunk;
+  });
+
+  req.on('end', () => {
+
+    fs.readdir('./public', 'utf8', (err, files) => {
+      if (!(files.includes(req.url.slice(1)))) {
+        const data = makeTemplate(body.elementName, body.elementSymbol, body.elementAtomicNumber, body.elementDescription);
+
+        fs.writeFile(`./public/${req.url.slice(1)}`, data, (err) => {
+          if (err) { throw err; }
```

Comment:
Same as the post handler, you don't want to throw an error here since it'll stall the server. Instead, res.write it back to the client with a 500 status